delatore/sources/influx.py

```diff
@@ -203,6 +203,175 @@ async def _get_status(self, metric):
                                last_time_ms.strftime(UNIFIED_TIME_PATTERN))
 
 
+class InfluxSourceLBTiming(InfluxSource):
+    """InfluxSourceLBTiming client"""
+
+    CONFIG_ID = 'influxdb_lb_timing'
+    _params: InfluxParams = None
+    list_elapsed_time = {}
+    list_hosts = []
+
+    _allowed_metrics = {'lb_timing', 'cpu_utilization', 'node_distribution', 'network_bytes_recv', 'network_bytes_send'}
+    _values_for_error_message = {'cpu_utilization', 'node_distribution', 'network_bytes_recv', 'network_bytes_send'}
+
+    _threshold = 60
+
+    @property
+    def threshold(self):
+        return self._threshold
+
+    @threshold.setter
+    def threshold(self, threshold):
+        self._threshold = threshold
+
+    async def init_list_elapsed_time(self):
+        for host in self.list_hosts:
+            self.list_elapsed_time[host] = [1, 1, 1, 1, 1]
+
+    async def get_update(self) -> dict:
+        last_time_ms, status = await self._get_status(self._metrics[0])
+        results = []
+        for host in self.list_hosts:
+            if (status == Status.FAIL):
+                results.append(generate_status(self._metrics[0].name, status,
+                                       last_time_ms.strftime(UNIFIED_TIME_PATTERN)))
+            else:
+                if min(self.list_elapsed_time[host]) > self._threshold:
+                    results.append(generate_error('InfluxdbLBTiming', await self._get_error_message(host)))
+        return generate_message(self.CONFIG_ID, results)
+
+    async def _get_error_message(self, host):
+        with open(_CONFIGS['error_message_lb_timing.txt'], 'r') as file:
+            str_template = file.read()
+        list_queries = [met.query.format(entity=met.metric_id) for met in self._metrics[1:]]
+        results_queries = await asyncio.gather(*[
+            self.influx_client.query(query) for query in list_queries
+        ])
+        results_queries_for_hosts = {self.list_hosts[0]: [], self.list_hosts[1]: []}
+        for result_query in results_queries:
+            for line in result_query.raw['series'][0]['values']:
+                results_queries_for_hosts[self.list_hosts[0]].append(line[1])
+            for line in result_query.raw['series'][1]['values']:
+                results_queries_for_hosts[self.list_hosts[1]].append(line[1])
+        str_template = str_template.format(
+                                           threshold=self.threshold,
+                                           current_response_time=self.list_elapsed_time[host][4],
+                                           hostname=host,
+                                           cpu_utilization=round(results_queries_for_hosts[host][0], 3),
+                                           network_bytes_recv=round(results_queries_for_hosts[host][2]/1000, 2),
+                                           network_bytes_send=round(results_queries_for_hosts[host][3]/1000, 2))
```

Comment:
Why series for different hosts is used?
`self.list_hosts[0]`, `self.list_hosts[1]`, `host`

Also, `KeyError`, `IndexError`